src/app/components/client/ProgressCard.tsx

```diff
@@ -133,16 +91,28 @@ export const ProgressCard = (props: Props) => {
           </div>
           <p>{l10n.getString("progress-card-resolved-by-you-headline")}</p>
         </div>
-
         <div className={styles.progressItem}>
           <div className={styles.progressStat}>
             <Image src={ExploringLaptopMinus} alt="" width="50" height="50" />
             <span>{props.autoRemoved}</span>
           </div>
           <p>{l10n.getString("progress-card-auto-removed-headline")}</p>
         </div>
+        {props.isPremiumUser && (
+          <div className={styles.progressItem}>
+            <div className={styles.progressStat}>
+              <Image
+                src={ExploringLaptopInProgress}
+                alt=""
+                width="50"
+                height="50"
```

Comment:
Looking at the SVG below, it seems to think it's 41&times;31.
Would this stretch the SVG, or just add some bonus whitespace around it and this is truly irrelevant?

Fun fact: each exploring-laptop-*.svg has _sliiightly_ different dimensions.

```sh
head -1 src/app/components/client/assets/exploring-laptop-*.svg

==> src/app/components/client/assets/exploring-laptop-check.svg <==
<svg width="41" height="32" viewBox="0 0 41 32" fill="none" xmlns="http://www.w3.org/2000/svg">

==> src/app/components/client/assets/exploring-laptop-in-progress.svg <==
<svg width="41" height="31" viewBox="0 0 41 31" fill="none" xmlns="http://www.w3.org/2000/svg">

==> src/app/components/client/assets/exploring-laptop-minus.svg <==
<svg width="40" height="31" viewBox="0 0 40 31" fill="none" xmlns="http://www.w3.org/2000/svg">
```